client-ts/signalr/README.md

```diff
@@ -3,7 +3,7 @@ JavaScript and TypeScript clients for SignalR for ASP.NET Core
 ## Installation
 
 ```bash
-npm install @aspnet/signalr-client
+npm install @aspnet/signalr
```

Comment:
I noticed line 39 refers to the package as `"@aspnet/signalR"`. Lower case the R?